resources/Kconif.diff

```diff
@@ -0,0 +1,5 @@
+84a85,88
+> config LOGO_NODEOS_CLUT224
+> 	bool "Gentoo-ised 224-colour logo"
```

Comment:
I didn't dare to change the name ^^

But nodeos-ized would sound pretty nice `^^